lib/cpu_registers.h

```diff
@@ -54,6 +54,11 @@ extern "C" {
 #define PQOS_MSR_ASSOC_QECOS_MASK  0xffffffff00000000ULL
 #define PQOS_MSR_ASSOC_RMID_MASK   ((1ULL << 10) - 1ULL)
 
+/*
+ * This is a global structure to store all the vendor defines
+ */
+extern struct pqos_vendor_config v_config;
```

Comment:
Add getter function to lib/cap.h
```suggestion
void _pqos_cap_get_vendor_config(struct pqos_vendor config *config)
```

Reply:
Taken care of your comments now. Removed the global structure and made it v_config static. Please review. thanks